extensions/amp-a4a/0.1/refresh-manager.js

```diff
@@ -46,6 +51,8 @@ const TAG = 'AMP-AD';
  * @param {!Window} win
  * @param {string} adType
  * @return {?number}
+ *
+ * @visibleForTesting
  */
 export function getPublisherSpecifiedRefreshInterval(element, win, adType) {
```

Comment:
Can't you get the adType from the element (element.getAttribute('type'))?

Reply:
Done.